core/include/arch/thread_arch.h

```diff
@@ -56,6 +57,14 @@
 char *thread_arch_stack_init(thread_task_func_t task_func, void *arg, void *stack_start, int stack_size);
 
 /**
+ * @brief   interrupt usage
```

Comment:
please try to be consistent: Though using a 'tabbed' indention of three spaces is the preferred way for @brief statements, this file indents only by 1 space... 

I would also prefer a more meaningful description like "Print information about the stack usage of the ISR stack"